src/components/CareerResourceList/CareerResourceList.js

```diff
@@ -32,163 +34,67 @@ const useStyles = makeStyles(theme => ({
-            <Table>
-              <TableHead>
-                <TableRow>
-                  <TableCell>Event</TableCell>
-                  <TableCell>Date/Time</TableCell>
-                  <TableCell>Description</TableCell>
-                  <TableCell>Link</TableCell>
-                </TableRow>
-                <TableRow>
-                  <TableCell>
-                    Career Fair
-                    <br></br>
-                  </TableCell>
-                  <TableCell>
-                    October 5th 2020 <br></br> 12-4 PM
-                  </TableCell>
-                  <TableCell>
-                    Network with recruiters from over 50 different companies.
-                    Formal dress code required.
-                  </TableCell>
-                  <TableCell>
-                    <a href="https://joinhandshake.com"> joinhandshake.com </a>
-                  </TableCell>
-                </TableRow>
-                <TableRow>
-                  <TableCell>Resume Review</TableCell>
-                  <TableCell>
-                    October 17th 2020 <br></br> 5-7 PM
-                  </TableCell>
-                  <TableCell>
-                    Meet up with a CST advisor to get a resume review in the
-                    Student Center!
-                  </TableCell>
-                  <TableCell>
-                    <a href="https://joinhandshake.com"> joinhandshake.com </a>
-                  </TableCell>
-                </TableRow>
-                <TableRow>
-                  <TableCell>Penn Hackathon</TableCell>
-                  <TableCell>
-                    October 23-24 2020 <br></br> ALL DAY
-                  </TableCell>
-                  <TableCell>
-                    Penn University is having its annual hackathon on October
-                    23-24. Be sure to sign up on Handshake by October 7th if you
-                    wish to attend.
-                  </TableCell>
-                  <TableCell>
-                    <a href="https://joinhandshake.com"> joinhandshake.com </a>
-                  </TableCell>
-                </TableRow>
-                <TableRow>
-                  <TableCell>Vanguard Networking Event</TableCell>
-                  <TableCell>
-                    October 26h 2020 <br></br> 12-3 PM
-                  </TableCell>
-                  <TableCell>
-                    Recruits from Vanguard will be visiting on campus in the
-                    SERC from 12-3 PM! Bring your resume!
-                  </TableCell>
-                  <TableCell>
-                    <a href="https://vanguard.com"> vanguard.com </a>
-                  </TableCell>
-                </TableRow>
-              </TableHead>
-            </Table>
-          </div>
-        </PerfectScrollbar>
-      </CardContent>
-      <CardActions className={classes.actions}></CardActions>
+const CareerEventList = props => (
+  <div className="careerEventList">
```

Comment:
Didn’t we want to make CareerResourceList it’s own component and get rid of CareerEventList?

Reply:
See other comment